myapp/app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class UsersController < ApplicationController
   helper_method :sort_direction
-
+  before_action :admin_user
```

Comment:
(`admin_user`だけユーザページを見れるの、前のステップのですかね？入れていただいてありがとうございます 🙌 
修正までは不要なのですが、`admin_user`というアクションだとどんな動作をするか想定しづらいので、自分だったらこう書くかなと思いました！ご参考まで 🙇 
```suggestion
  before_action :redirect_to_root_page, if: :admin?
  
  def redirect_to_root_page
      redirect_to root_url
  end
```
)